### PR TITLE
Give and revoke multiple permissions

### LINKED ISSUE #572: Return type does not allow Collection, and 'guard_name' condition uses "whereId" instead of "whereIn"
https://github.com/spatie/laravel-permission/blob/03eb748141432b5d94eb1613b99f818aac1f5070/src/Traits/HasPermissions.php#L73

HasPermissions::getStoredPermission supports multiple permissions as an input, but is tied to a single permission for the return type.

Also, if $permissions is an array, there's a call to "whereId" instead of "whereIn", which I'm pretty sure is generating an invalid query:

Original query:
```sql
select * from `permissions` where `name` in ("test-permission") and `id` = "guard_name";`
```

Updated query:
```sql
select * from `permissions` where `name` in ("test-permission") and `guard_name` in ("web");
```

I've prepared [a fix](https://github.com/artissant/laravel-permission/tree/feature/give-and-revoke-multi-permissions) but thought I'd create an issue before making a pull request. This _shouldn't_ introduce any breaking changes because passing an array/Collection to revokePermissionTo would currently trigger an error.

### PR DESCRIPTION
**Pull request by artissant**
Fixes #572

### CODE REVIEW

File: src/Traits/HasPermissions.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ public function syncPermissions(...$permissions)
     /**
      * Revoke the given permission.
      *
-     * @param \Spatie\Permission\Contracts\Permission|string $permission
+     * @param Permission|Permission[]|string|string[] $permission
```
Comment (AlexVanderbist) [inline src/Traits/HasPermissions.php]: Can we use the FQCN here?

File: tests/PermissionTest.php [hunk 2]
```diff
@@ -41,4 +41,16 @@ public function it_has_user_models_of_the_right_class()
         $this->assertTrue($this->testUserPermission->users->first()->is($this->testUser));
         $this->assertInstanceOf(User::class, $this->testUserPermission->users->first());
     }
+
+    /** @test */
+    public function it_can_give_and_revoke_multiple_permissions()
```
Comment (AlexVanderbist) [inline tests/PermissionTest.php]: Can we move this test to the `HasPermissionsTest`? I think that'd be a more appropriate context

File: src/Traits/HasPermissions.php [hunk 3]
```diff
@@ -68,9 +68,9 @@ public function revokePermissionTo($permission)
     /**
      * @param string|array|\Spatie\Permission\Contracts\Permission|\Illuminate\Support\Collection $permissions
      *
-     * @return \Spatie\Permission\Contracts\Permission
+     * @return Permission|Permission[]|\Illuminate\Support\Collection
```
Comment (AlexVanderbist) [inline src/Traits/HasPermissions.php]: FQCN as well :)

**Comment by artissant** [comment]
Hi, nice one for looking over it, I've pushed those tweaks up now.

**Comment by drbyte** [comment]
Thanks!

**Pull request merged**